src/ansiblelint/__main__.py

```diff
@@ -141,7 +141,7 @@ def _do_transform(result: "LintResult", opts: "Namespace") -> None:
     from ansiblelint.transformer import Transformer
 
     # future: maybe pass options to Transformer
-    transformer = Transformer(result)
+    transformer = Transformer(result, options.write_list)
```

Comment:
How about making it receive the entire options, as we might want to add other options that control the transformation soon. That should make it future proof.

Reply:
done